app/views/groups/index.html.erb

```diff
@@ -49,33 +49,7 @@
     <%= link_to t('upload'), 'javascript:void(0);', onclick:'modal_upload.open();' %>
 <% end %>
 
-<div id="notice">
-  <div class="success" id="add_success" style="display: none;"></div>
-  <% # This is for error display injected by JavaScript %>
-
-  <div class="error" id="errors" style="display: none; ">
```

Comment:
I go through all the JS file in the project but I can't find any code related to this div tag. I think the CSS is combined with classes which is used for new flash messages as well so I didn't delete it.